server/package.json

```diff
@@ -38,7 +38,8 @@
   "license": "Apache-2.0",
   "nodemonConfig": {
     "ignore": [
-      "test/*"
+      "test/*",
+      "persist/*"
```

Comment:
persist?

Reply:
Following previous explanation about persist folder, it avoids nodemon server reload loop on bridge configuration (when HomeKit integration is starting it write files in this folder)